testsuite/single_node_performance.py

```diff
@@ -58,12 +58,12 @@ class RunGroupConfig:
 
 
 SELECTED_FLOW = Flow[os.environ.get("FLOW", default="LAND_BLOCKING")]
-IS_PREVIEWNET = SELECTED_FLOW in [Flow.PREVIEWNET, Flow.PREVIEWNET_LARGE_DB]
+IS_MAINNET = SELECTED_FLOW in [Flow.MAINNET, Flow.MAINNET_LARGE_DB]
 
 DEFAULT_NUM_INIT_ACCOUNTS = (
-    "100000000" if SELECTED_FLOW == Flow.PREVIEWNET_LARGE_DB else "2000000"
+    "100000000" if SELECTED_FLOW == Flow.MAINNET_LARGE_DB else "2000000"
 )
-DEFAULT_MAX_BLOCK_SIZE = "25000" if IS_PREVIEWNET else "10000"
+DEFAULT_MAX_BLOCK_SIZE = "25000" if IS_MAINNET else "10000"
```

Comment:
This is too big for mainnet,  right?

Reply:
We can keep the same benchmark as previewnet for hardware evaluation purpose for now - we can work on tuning the block size later.

Reply:
For evaluation sure, but then it will be harder to track why we are not reaching those numbers in prod.
We anyways should recalibrate numbers if performance changes in the meantime?

But recheck that long mode is still working.

Reply:
Yes, I am in the process of rerunning the Forge benchmarks fine tuning the configurations like block size so that we can ship them to mainnet. Once we get reasonable latency without trading off throughput, we can finalize them and also update this benchmark.